### ISSUE TITLE
TreetableCheckbox & TableCheckbox allow aria-attributes from outside (same for radio-Buttons please)

### ISSUE THREAD

**Issue by B34v0n** [primefaces/primeng#18967]
### Describe the bug

It's not possible to give the treetable-checkbox or table-checkbox some aria attributes.

For the Checkbox itself it is possible:
https://github.com/primefaces/primeng/blob/master/packages/primeng/src/checkbox/checkbox.ts#L105-L110
it's possible to define an ariaLabel and an ariaLabelledBy.

It should be possible to set these inputs from outside and they should go through to the real p-checkbox.
Otherwise it is impossible to set a label to the checkbox in tables.
See https://github.com/primefaces/primeng/blob/9255fd440774261acc3f22e405f2525963b77841/packages/primeng/src/treetable/treetable.ts#L3262 and https://github.com/primefaces/primeng/blob/9255fd440774261acc3f22e405f2525963b77841/packages/primeng/src/table/table.ts#L4971

This way the checkbox  cannot be read by screenreaders as what item it selects.

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/jt7kxoad?file=package.json

### Environment

All Environments. This change is important to be compliant with accessibility.

### Angular version

^20.0.4

### PrimeNG version

v20

### Node version

any

### Steps to reproduce the behavior

1. open stackblitz or https://primeng.org/treetable#checkbox
2. inspect a checkbox
3. See that the checkbox is just an input without any description

### Expected behavior

The checkbox should be describable with aria elements.

**Comment by B34v0n** [comment]
This issue is still relevant
